packages/jumpstarter-driver-corellium/jumpstarter_driver_corellium/corellium/api_test.py

```
@@ -0,0 +1,183 @@
+import os
+
+import pytest
+import requests_mock
+
+from .types import *
```

Ruff (0.8.2)

`from .types import *` used; unable to detect undefined names

(F403)